test/functional/tests/common/lvm_common_utils.py

```diff
@@ -29,7 +30,8 @@ def run_fio_on_lvm(volumes: []):
 
 
 def get_test_configuration():
-    config = InitConfig.create_init_config_from_running_configuration()
+    InitConfig.create_init_config_from_running_configuration()
+    config_output = TestRun.executor.run(f"cat {opencas_conf_path}")
     devices = get_block_devices_list()
 
-    return config, devices
+    return config_output.stdout, devices
```

Comment:
why is this better then the old version?

Reply:
Because comparing config on test level was no longer working - I suspect that config format changed: now we have inside cache config lines table and core config lines table. Parsing saved config file this way and using ".stdout" allows for simple comparison in test (comparison of configuration before and after reboot)